Recruit-Training/Telemetry-Recruit-Training/.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
```

Comment:
There's already a `.travis.yml` file in the root directory, so the build will actually run that one. Try modifying that one.